english/hands/door/door.md

```diff
@@ -1,5 +1,6 @@
 After having watched MacGyver in your childhood, you are able to build up a door with the rope by kindle the fire
 with the lighter.
 
-Finally, the marshmallows start to melt down and you see a beautiful beach with two moons in the sky. You have travel
-through space and time without knowing it!
\ No newline at end of file
+Finally, the marshmallows start to melt down and you see a beautiful beach with two moons in the sky. You have travel through space and time without knowing it!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.